tests/src/test/java/org/neo4j/graphalgo/impl/KShortestPathsUserTest.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2017 "Neo4j, Inc." <http://neo4j.com>
+ * <p>
+ * This file is part of Neo4j Graph Algorithms <http://github.com/neo4j-contrib/neo4j-graph-algorithms>.
+ * <p>
+ * Neo4j Graph Algorithms is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphalgo.impl;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.neo4j.graphalgo.TestProgressLogger;
+import org.neo4j.graphalgo.api.Graph;
+import org.neo4j.graphalgo.core.GraphLoader;
+import org.neo4j.graphalgo.core.heavyweight.HeavyGraph;
+import org.neo4j.graphalgo.core.heavyweight.HeavyGraphFactory;
+import org.neo4j.graphalgo.impl.yens.WeightedPath;
+import org.neo4j.graphalgo.impl.yens.YensKShortestPaths;
+import org.neo4j.graphdb.Direction;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.internal.kernel.api.exceptions.KernelException;
+import org.neo4j.test.rule.ImpermanentDatabaseRule;
+
+import java.util.List;
+import java.util.function.DoubleConsumer;
+
+import static org.mockito.Mockito.mock;
+
+/**
+ * @author mknblch
+ *
+ * TODO rm if verified
+ */
+@Ignore
+public class KShortestPathsUserTest {
+
+    private static final String PATH = "/Users/mknobloch/Downloads";
```

Comment:
hardcoded path to directory, are you going to remove this per comment as I understand?

Reply:
@kcrimson yeh that's only been left there temporarily as this is WIP and I've been testing it with your dataset before we merge the PR in. We won't merge it in with that hardcoded path or the build won't work :)